src/Okta.Sdk/Internal/CollectionAsyncEnumerator{T}.cs

```diff
@@ -82,15 +82,15 @@ public sealed class CollectionAsyncEnumerator<T> : IAsyncEnumerator<T>
         private static HttpRequest GetNextLink(HttpResponse response)
         {
             var linkHeaders = response
-                .Headers
+                .Headers?
                 .Where(kvp => kvp.Key.Equals("Link", StringComparison.OrdinalIgnoreCase))
                 .Select(kvp => kvp.Value);
 
-            var nextUri = LinkHeaderParser
+            var nextUri = (linkHeaders != null) ? LinkHeaderParser
                 .Parse(linkHeaders.SelectMany(x => x))
                 .Where(x => x.Relation == "next")
                 .SingleOrDefault()
-                .Target;
+                .Target : null;
```

Comment:
Use `?.Target` in case `SingleOrDefault` returns null

Reply:
`WebLink` is a `struct` so is not applicable